packages/flutter_tools/test/host_cross_arch.shard/macos_content_validation_test.dart

```diff
@@ -13,20 +13,30 @@ import '../src/common.dart';
 import '../src/darwin_common.dart';
 
 void main() {
+  final String flutterBin = fileSystem.path.join(
+    getFlutterRoot(),
+    'bin',
+    'flutter',
+  );
+
+  setUpAll(() {
+    processManager.runSync(<String>[
+      flutterBin,
+      'config',
+      '--enable-macos-desktop',
```

Comment:
Can we do it for all tool shards? instead of per test?
https://github.com/flutter/flutter/blob/0d087562320d0d28293b6bd794e9fdd418038698/dev/bots/test.dart#L587

Reply:
Actually it doesn't matter, if we see it again we can hoist it up.